go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/mongodb/mongodb-atlas-cli
 
-go 1.20
+go 1.21.3
```

Comment:
Should we be tied to a minor version?

Reply:
No strong opinion. What do you do in kubernetes operator?

For context 
https://go.dev/blog/toolchain

Reply:
I have used toolchain keyword and downgraded to 1.20 without patch

Reply:
I see... I think there is no concern here.